control-plane/PROJECT

```diff
@@ -126,4 +126,20 @@ resources:
   kind: RouteTimeoutFilter
   path: github.com/hashicorp/consul-k8s/control-plane/api/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1beta1
+    namespaced: true
+  domain: hashicorp.com
+  group: consul
+  kind: HTTPRouteAuthFilter
```

Comment:
this too even though I'm fairly certain this file is almost entirely unused

Reply:
I _think_ this file is autogenerated, let me make sure I didn't miss any commands to generate this

Reply:
updated this